spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -715,7 +715,6 @@
       expect(current_path).to eq(jobseekers_job_preferences_step_path(:roles))
       expect(page).to have_css("h2", text: "There is a problem")
 
-      first("label", text: "Teacher", exact_text: true).sibling("input").set(true)
```

Comment:
Why are we removing this?

Reply:
It's an anomaly caused by the error state. It obfuscates the test for no obvious reason other than an apparent desire to choose 'Teacher' rather than any of the other selections